lib/umbra/publisher.rb

```diff
@@ -1,33 +1,30 @@
 module Umbra
-  class Publisher < SynchronousPublisher
-    MAX_QUEUE_SIZE = 100
+  class Publisher
+    DEFAULT_MAX_QUEUE = 100
+    DEFAULT_MIN_THREADS = 1
+    DEFAULT_MAX_THREADS = 1
```

Comment:
i wonder if this should be in config/env vars to be more useful

Reply:
Not sure what you mean by this, this change enables users to configure all of these via
```ruby
Umbra.configure do |config|
  config.publisher = Umbra::Publisher.new(max_treads: ENV['UMBRA_MAX_THREADS'])
end
```

Do you mean having the ability to configure this purely via passing an options hash to `Umbra.configure`? I guess something like that would be similar to how `Rails.cache` is configured 🤔

Reply:
nope, misread it. all good 👍